assets/extensions/examples/online-extensions.css

```diff
@@ -0,0 +1,6 @@
+/*
+  Customize the logo in the homepage header
+*/
+.navbar-pf-alt #header-logo {
```

Comment:
I think we should update our style to be less specific for extensions that want to change it (and make sure we only use `#header-logo` for the navbar).